SpatialGDK/Extras/schema/core_types.schema

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 package unreal;
 
+type LoadInfo {
+    string path = 1;
+    bool no_load = 2;
+}
+
 type UnrealObjectRef {
     EntityId entity = 1;
     uint32 offset = 2;
-    option<string> path = 3;
+    option<LoadInfo> load_info = 3;
```

Comment:
I like the addition of this new type but I think it's more important to make the C++ type match the schema type in layout. If we're not going to be grouping the `path` and `no_load` in C++ I don't think we should do it in schema.